tests/acceptance/_bootstrap.php

```diff
@@ -1 +1,3 @@
 <?php
+define("TIMEOUT", 60);
```

Comment:
I would also add a /tests/readme.md file explaining the particular details of the tests. In this case you can say:

- you can modify the timeout time by changing the value of this constant... lower for fast machines and higher for slow computers...